pwiz_tools/Skyline/Properties/Settings.Designer.cs

```diff
@@ -1,3192 +1,3217 @@
-﻿//------------------------------------------------------------------------------
```

Comment:
The diffs for the settings files are too big. They should only impact the settings you changed.

Reply:
Reverted the whitespace changes.